sdk/python/tests/test_feature_store.py

```diff
@@ -72,6 +72,24 @@ def feature_store_with_gcs_registry():
     )
 
 
+@pytest.fixture
+def feature_store_with_s3_registry():
+    import boto3
+
+    s3 = boto3.resource("s3")
+    bucket_name = f"feast-registry-test-{int(time.time() * 1000)}"
```

Comment:
Let's use `feast-integration-tests` bucket, which I have already created in Feast's AWS account for integration tests.

Reply:
done